ideas.html

```diff
@@ -3,6 +3,19 @@
 <html>
     <head>
         <meta charset="utf-8">
+        <style>
```

Comment:
@jasper-gruhl @lilygruhl this adds a background image to the whole body and adds styling to any tag that contains the class conveyor-belt.